_episodes/04-dask-abstractions.md

```diff
@@ -115,10 +115,10 @@ bag.filter(pred).compute()
 ### Reduction
 
 ~~~python
-bag.reduction(sum, sum).visualize()
+bag.reduction(f, sum).visualize()
```

Comment:
Please read: https://docs.dask.org/en/latest/_modules/dask/bag/core.html#Bag.reduction
The bag is split in chunks, each chunk to be processed separately using function one, then these results are aggregated using the second function. If we compute a sum, then both functions should be sum.

Reply:
Oh, I see. Then we can undo commit https://github.com/carpentries-incubator/lesson-parallel-python/pull/47/commits/bd851b549cee61f50f5eff0c720929ddc8da2759

Reply:
Commit https://github.com/carpentries-incubator/lesson-parallel-python/pull/47/commits/c05baaf22ff747953c5c6cd30be51e4b708451d1 reverts the problematic commit